web-client/src/models/offers/offersWithLocation.ts

```diff
@@ -9,7 +9,7 @@ import { IUserAddress } from '../users/privilegedInformation';
 import { IOffer, Offer, OfferStatus } from './index';
```

Comment:
I had to make changes to address or I got an error.  This needs to be fixed in 3.1, as you currently can't specify an address when making an offer.  I would wait to 3.1 to do this, as there has to be some changes to the data model then.  To keep track, I would add a bug.